bin/runner.go

```diff
@@ -55,40 +53,54 @@ func main() {
 		// derive the required field from the experiment & engine and set into experimentDetails struct
 		if err := experiment.SetValueFromChaosResources(&engineDetails, clients); err != nil {
 			log.Errorf("Unable to set values from Chaos Resources, error: %v", err)
-			recorder.ExperimentSkipped(experiment.Name, utils.ExperimentNotFoundErrorReason)
+			msg := "Experiment Job creation failed, skipping Chaos Experiment: " + experiment.Name
```

Comment:
I think we can keep these messages pre-build?

Reply:
We have a few dynamic variables inside the message so I suppose this approach will provide more freedom then pre-build